website/content/ko/wgs/platforms/whitepaper/index.md

```diff
@@ -1,505 +1,217 @@
 ---
-title:  "CNCF 플랫폼 백서"
+제목:  "CNCF 플랫폼 백서(White Paper)"
```

Comment:
It seems that the key here needs to stay in English, that is it should be "title:"

I say this cause in the current deployment preview no title is rendered for this section in the Korean version.

Reply:
I fixed this!